batch/README.md

```diff
@@ -17,20 +24,40 @@ brew cask reinstall minikube # or equivalent on your OS
 minikube start
 ```
 
-When you want to return to using a google k8s cluster, you can run this:
+To use a remote Google Cloud k8s cluster instead:
 
+1. Login with a Google account that is linked to a Google Cloud project that manages the cluster of interest
+```sh
+gcloud auth login
 ```
-gcloud container clusters get-credentials CLUSTER_NAME
+
+2. Set the active project 
+```sh
+gcloud projects list
```

Comment:
Let's add a `#` or `$` to give a visual indication of which lines are command and which are output. I realize this makes copy-paste mildly more annoying. In other parts of the README, we've neglected to include output. I find the output useful for understanding here.

Reply:
Should be better